pydoc/getting_started/introduction/about.md

```diff
@@ -0,0 +1,48 @@
+# About DLite
```

Comment:
Documentation markdown files should be placed in doc/

Reply:
As far as I understand, @quaat is doing the merge of `doc/` and `pydoc/` and suggested to update `pydoc/` until `doc/` is empty and then rename `pydoc/` to `doc/`?